arrow-continuations/src/test/kotlin/generic/SuspendingComputationTest.kt

```diff
@@ -125,26 +119,6 @@ class SuspendingComputationTest : StringSpec({
     } shouldBe Left("test")
   }
 
-  "Short-circuiting cancels Arrow Fx Coroutines" {
```

Comment:
This is covered by the KotlinX Coroutines test now that Arrow Fx Coroutines uses the KotlinX Coroutines runtime.